telnyx/util.py

```diff
@@ -18,6 +18,9 @@
 
 __all__ = ["io", "parse_qsl", "utf8", "log_info", "log_debug", "logfmt"]
 
+# Special valid characters in telnyx id's objects. Format: `=/$#`
+telnyx_valid_id_parts = "=-"
```

Comment:
`-` should always be OK, unescaped or no
`=` "should" always be escaped (according to [RFC-3986](https://tools.ietf.org/html/rfc3986)) unless it's actually part of a query string, but servers generally seem to be OK with it. I'm not sure if in this SDK it'd be escaped twice, once that we bypass using this, and again by the actual library (requests, urlfetch, etc.).